gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/SlackController.java

```diff
@@ -47,13 +47,15 @@
 
   private final SlackConfigProperties slackConfigProperties;
   private final SlackService slackService;
+  private final Integer refreshDelay;
```

Comment:
Not sure if this needs to be a thing since you are just reading out: `slack.channelRefreshIntervalInMs`